plenum/test/node_catchup_with_3pc/test_stashing_checkpoints_after_view_change.py

```diff
@@ -71,7 +71,11 @@ def test_checkpoints_after_view_change(tconf,
                                       sdk_wallet_client,
                                       num_reqs)
             looper.run(
-                eventually(check_last_ordered_3pc_on_all_replicas, rest_nodes,
+                eventually(check_last_ordered_3pc_on_master, rest_nodes,
+                           (1, num_reqs + 1))
+            )
+            looper.run(
+                eventually(check_last_ordered_3pc_on_backup, rest_nodes,
```

Comment:
Why last ordered is different on master and backups?

Reply:
because additional master 3pc batch was sent because of view change